acceptance/testdata/secret/secret-org.txtar

```diff
@@ -0,0 +1,16 @@
+# Prepare organization secret, GitHub Actions uppercases secret names
+env2upper ORG_SECRET_NAME=$RANDOM_STRING
+
+# Confirm organization secret does not exist, will fail admin:org scope missing
+! exec gh api /orgs/$ORG/actions/secrets/$ORG_SECRET_NAME
```

Comment:
> Can you talk to me a bit about why you feel you need to check a secret with this name doesn't exist before proceeding? How's this moving the needle on your confidence levels?

I had 2 concerns initial writing this test:

1. Organization resources don't have a neat container like a repository that can be cleaned up when test completes
2. The permissions needed for working with organization secrets and variables might be uncommon to token used for testing

So I thought confirming that the random string I'm about to use for testing didn't exist as a way to ensure no prior state would conflict and the token has sufficient privileges before moving on.

Reply:
Interesting... I am doing something similar in my PR. I've dropped a [comment](https://github.com/cli/cli/pull/9783/files#r1805452914) there to have a tightly scoped discussion on it, because ultimately I think @andyfeller's reasoning here is sound but isn't relevant to my case, which still falls within the scope of @williammartin's question.

Reply:
Thanks! I responded over on Tyler's PR as well about why I think a pre-assertion could be valuable in my eyes.

> Organization resources don't have a neat container like a repository that can be cleaned up when test completes

It feels very similar to `repo create` commands, where we don't check that a repo with a `$RANDOM_STRING` name doesn't exist before creating it.

> The permissions needed for working with organization secrets and variables might be uncommon to token used for testing

That's an interesting point. I have ideas around making scopes requirements clear that would remove the need for this.

How would you feel if each script started with a command like:

```
scopes admin:org,repo,... 
```

Such that each script enumerated the scopes required to continue with the test? It feels like it would be very informative to the reader.